### PR TITLE
Account deactivation on Riot Android

### PR DESCRIPTION
**Pull request by bmarty**
Account deactivation is now managed on the Android application

This PR also include:
- build script enhancement (along with Jenkins configuration change)

- creation of a DebugReceiver, to help debugging the application. It is only activated on DEBUG versions. Current supported actions are:
    - `ls` on the application filesystem
    - dump of the SharedPref.

### CODE REVIEW

**Comment by bmarty** [comment]
Note: https://github.com/matrix-org/matrix-android-sdk/pull/275 is needed, so Travis will fail :(

File: vector/src/main/java/im/vector/activity/CommonActivityUtils.java [hunk 1]
```diff
@@ -427,6 +428,77 @@ public void onSuccess(Void info) {
...
+
+                try {
+                    ShortcutBadger.setBadge(context, 0);
+                } catch (Exception e) {
+                    Log.d(LOG_TAG, "## logout(): Exception Msg=" + e.getMessage());
+                }
+
+                // warn that the user logs out
+                Collection<MXSession> sessions = Matrix.getMXSessions(context);
+                for (MXSession session : sessions) {
```
Comment (manuroe) [inline vector/src/main/java/im/vector/activity/CommonActivityUtils.java]: We could make that only for `mxSession`

Comment (bmarty) [reply vector/src/main/java/im/vector/activity/CommonActivityUtils.java]: ok, yes, done

**Pull request merged**